_posts/2021-11-18-real-time-and-historical-ad.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Anomaly Detection for Historical Data and Real-time Streaming"
+authors:
+  - ohltyler
+  - bpavani
```

Comment:
Pavani doesn't have a picture on her bio - it would be great if she had one.

Reply:
Discussed with Pavani, she suggested to not be added as an author actually, so I've removed.